what

```diff
@@ -117,12 +126,12 @@ for tty, st, cmds in ttys:
                        pretty_time(st.st_ctime), \
                        pretty_time(st.st_atime), \
                        pretty_time(st.st_mtime), cmd)
-        print color + s[:cols] + "\033[0m"
+        print(color + s[:cols] + "\033[0m")
 
 logged_in_uids = set()
 for tty in ttys:
     logged_in_uids.add(tty[1].st_uid)
-for uid, count in notty.items():
+for uid, count in list(notty.items()):
```

Comment:
Similar point here as above: you don't need the `list` since you're iterating over the items anyway.